pkg/controllers/clusterresourceplacementeviction/controller_test.go

```diff
@@ -1513,7 +1507,7 @@ func TestReconcileForIncompleteEvictionMetric(t *testing.T) {
 		t.Errorf("reconcile should have failed")
 	}
 
-	metricFamilies, err := customRegistry.Gather()
+	metricFamilies, err := ctrlmetrics.Registry.Gather()
```

Comment:
I assume that this is not related to the reportDiff change? Where was FleetEvictionStatus registered?

Reply:
When I write the UT, I realized this metrics UT needs fix. I fixed for both updateRun and eviction.